lib/engine/action/sell_company.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require_relative 'base'
+
+module Engine
+  module Action
+    class SellCompany < Base
+      attr_reader :entity, :price
```

Comment:
you don't need company?

Reply:
No, entity is the company. entity.owner is the player who is selling.

Reply:
i don't think this is how actions are generally done.

the entity is the one doing the action ( the entity.owner) and then you need a target, which is the company

Reply:
I was just copying how the buy_shares action is used during an exchange ability on a private company. entity is the company and entity.owner is the player in that case too.

Reply:
i think in buy_shares entity is the person buying the share, and shares refers to the shares you are buying?